frame/vesting/src/lib.rs

```diff
@@ -175,7 +175,7 @@ pub mod pallet {
 					starting_block: begin
 				});
 				let reasons = WithdrawReasons::TRANSFER | WithdrawReasons::RESERVE;
-				T::Currency::set_lock(VESTING_ID, who, locked, reasons);
+				T::Currency::set_lock(VESTING_ID, who, locked, reasons).expect("problem adding lock in vesting genesis");
```

Comment:
It's legal to potentially panic while building genesis?

Reply:
yes we consider it ok. if genesis fail to build there is no harm, the chain doesn't start.